optaweb-vehicle-routing-docs/src/main/asciidoc/quickstart.adoc

```diff
@@ -0,0 +1,175 @@
+
+.Procedure:
+. To verify the current Java installation, enter the following command:
++
+[source,shell]
+----
+java -version
+----
+. If necessary, install OpenJDK 8.
+* To install OpenJDK 8 on Fedora, enter the following command:
++
+[source,shell]
+----
+sudo dnf install java-1.8.0-openjdk-devel
+----
+* To install OpenJDK on other platforms, follow instructions at https://openjdk.java.net/install/.
+
+=== Download OptaWeb Vehicle Routing
+
+Download the OptaWeb Vehicle Routing distribution archive, available from the OptaPlanner website, to quickly evaluate OptaWeb Vehicle Routing without having to set up build tools.
+
+.Prerequisites
+* Java 8 or higher is installed.
+
+// TODO cross-reference
+NOTE: If you want to modify OptaWeb Vehicle Routing and build it yourself or contribute to upstream, see the Development chapter.
+
+.Procedure
+. Go to https://www.optaplanner.org/download/download.html
+and click the *OptaWeb Vehicle Routing* tab.
+. Click *Download OptaWeb Vehicle Routing <version>*.
++
+.OptaPlanner download page
+image::download.png[OptaPlanner download page at www.optaplanner.org,salign="center"]
++
+. Extract the downloaded distribution ZIP file. The archive contains source files and a binary build of OptaWeb Vehicle Routing as well as the OptaWeb Vehicle Routing documentation.
++
+.Content of the OptaWeb Vehicle Routing distribution archive
+image::distribution.png[Content of the OptaWeb Vehicle Routing distribution archive,align="center"]
+
+=== Run OptaWeb Vehicle Routing
+
+After you download OptaWeb Vehicle Routing, use the `run.sh` script to run it.
+
+.Prerequisites
+* Internet access is available.
+When OptaWeb Vehicle Routing runs it uses third-party public services such as link:https://www.openstreetmap.org/about[OpenStreetMap] to display map tiles and provide search results.
+* Java 8 or higher is installed.
+* A directory called `openstreetmap` with at least one OSM file exists.
+* A country code to use in search queries is identified.
+
+.Procedure
+Enter the following command, where `<DATA_DIR>` is the parent of the `openstreetmap` directory, `<COUNTRY>` is the country that you downloaded data for and `<CODE>` is the code for that country:
+
+[source,bash]
+----
+./bin/run.sh <DATA_DIR> <COUNTRY>-latest.osm.pbf <CODE>
+----
+
+For example, if you want to run OptaWeb Vehicle Routing with Belgium and you downloaded the `belgium-latest.osm.pbf` from the Geofabrik website and saved it as `~/.vrp/openstreetmap/belgium-latest.osm.pbf`, enter the following command:
+
+[source,bash]
+----
+./bin/run.sh ~/.vrp/ belgium-latest.osm.pbf BE
+----
+
+=== Using air distance mode
+
+OptaWeb Vehicle Routing can work in air distance mode that calculates travel times based on the distance between two coordinates.
+Use this mode in situations where you need to get OptaWeb Vehicle Routing up and running as quickly as possible and do not want to use an OSM (OpenStreetMap) file.
```

Comment:
So because this content is not modularized you only need to define acronyms once, on the first instance.